connection/src/thick.rs

```diff
@@ -126,6 +130,7 @@ impl AttestationError for ThickClientAttestationError {
             Self::Ake(AkeError::AttestationEvidenceVerification(_)) => false,
             Self::Ake(_) => true,
             Self::InvalidResponderID(_, _) | Self::UriConversionError(_) => false,
+            Self::Other(_) => false,
```

Comment:
Since the time errors should only occur if the system time is moved backwards, it might seem like this should be true, but since the variant is just `Other`, it may be used for other things in the future, so no assumptions can be made about whether or not to retry.